frontend/src/index.tsx

```diff
@@ -1,17 +1,32 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import Post from './component/Post';
 import './index.css';
-import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <div className='Center'>
+      <Post
+        name={'Alexander Peterson'}
+        time={'4:44'}
+        content={'Pommes und Disco'}
+        likes={420}
+        dislikes={69}
+        isComment={false}
+      />
+      <Post
+        name={'Alexander Peterson'}
+        time={'4:44'}
+        content={'Pommes und Disco'}
+        likes={420}
+        dislikes={69}
+        isComment={true}
+      />
+    </div>
   </React.StrictMode>,
   document.getElementById('root')
 );
 
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```

Comment:
Why did you remove this line?

Reply:
I cant remember removing it. Probably a mistake.